dbschema/2.views.sql

```diff
@@ -1,37 +1,53 @@
-CREATE MATERIALIZED VIEW mvw_user_registry_first_ownerships AS (
+CREATE MATERIALIZED
+VIEW mvw_user_registry_first_ownerships AS
+(
     SELECT tbl_token_transfers.registry_address, tbl_token_transfers.to_address AS owner_address, min(tbl_blocks.block_date) AS join_date
-    FROM tbl_blocks
+FROM tbl_blocks
     JOIN tbl_token_transfers ON tbl_token_transfers.block_number = tbl_blocks.block_number
-    WHERE tbl_token_transfers.registry_address = ANY(array(select registry_address from tbl_gallery_customers))
+WHERE tbl_token_transfers.registry_address = ANY(array
+(select registry_address
+from tbl_gallery_customers)
+)
     GROUP BY tbl_token_transfers.registry_address, tbl_token_transfers.to_address
 );
 CREATE UNIQUE INDEX mvw_user_registry_first_ownerships_registry_address_owner_address ON mvw_user_registry_first_ownerships (registry_address, owner_address);
 CREATE INDEX mvw_user_registry_first_ownerships_registry_address ON mvw_user_registry_first_ownerships (registry_address);
 
 
-CREATE MATERIALIZED VIEW mvw_user_registry_ordered_ownerships AS (
+CREATE MATERIALIZED VIEW mvw_user_registry_ordered_ownerships AS
```

Comment:
whats going on here, why have you changed this?

Reply:
must have been an issue with merging, I will change it back